Microsoft.Toolkit.Uwp.SampleApp/SamplePages/BladeView/BladePage.xaml

```diff
@@ -26,20 +21,20 @@
                                 IsOpen="True"
                                 TitleBarVisibility="Collapsed">
                 <StackPanel Margin="24">
-                    <TextBlock Text="This first blade has a hidden TitleBar, so you can't close it."
```

Comment:
I suppose that's because of the different merges I made during the PR. What do you want to revert in particular ?